roles/login_ocp/tasks/main.yml

```diff
@@ -4,6 +4,4 @@
    msg: "Logging into cluster using KUBECONFIG: {{ ansible_env.KUBECONFIG }}"
 
 - name: login to cluster
-  vars:
-    kubeadmin_password: "{{ ansible_env.kubeadmin_password }}"
-  shell: oc login -u admin -p {{ kubeadmin_password }}
+  shell: "oc login -u {{ user }} -p {{ pass }}"
```

Comment:
kubeadmin_password is an env var, there is no need to keep password in a config anymore. But you can prompt login.